frontend/compute-system/src/themes/mainTheme.ts

```diff
@@ -39,6 +41,9 @@ const mainTheme = createMuiTheme({
     white: {
       main: "#FFFFFF",
     },
+    sLight: {
```

Comment:
if this color refers to stars which are highligthed then it would be better to name it `starsLitColor` or sth similar
if this a light version of some color then:
```
<colorName>: {
    light: <color>
}